src/applications/vaos/services/referral/index.js

```diff
@@ -28,6 +28,27 @@ export async function getProviderById(providerId) {
   return response.data;
 }
 
+export async function postReferralAppointment({
+  referralId,
+  slotId,
+  draftApppointmentId,
+}) {
+  const response = await apiRequestWithUrl(
+    `/vaos/v2/epsApi/appointments/${referralId}`,
```

Comment:
We should probably move the referralId to the body to avoid exposing that number in Google analytics